recipes/poppler/all/conanfile.py

```diff
@@ -84,32 +101,35 @@ def configure(self):
             del self.options.with_libiconv
         if self.options.fontconfiguration == "win32" and self.settings.os != "Windows":
             raise ConanInvalidConfiguration("'win32' option of fontconfig is only available on Windows")
-        if self.settings.compiler == "gcc":
-            if tools.Version(self.settings.compiler.version) < 5:
-                raise ConanInvalidConfiguration("poppler requires at least gcc 5")
-        elif self.settings.compiler == "Visual Studio":
-            if tools.Version(self.settings.compiler.version) < 15:
-                raise ConanInvalidConfiguration("poppler requires at least Visual Studio 15 (2017)")
+        
+        # C++ standard required
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 14)
```

Comment:
```suggestion
            tools.check_min_cppstd(self, self._cppstd_required)
```

Reply:
This library (`poppler`) requires 14 to build, not 17. Only if using Qt6 you will require 17, but the Qt recipe will raise the ConanInvalidConfiguration then.